addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -111,6 +111,10 @@ var KanbanRenderer = BasicRenderer.extend({
         var templates = findInNode(this.arch, function (n) { return n.tag === 'templates';});
         transformQwebTemplate(templates, state.fields);
         this.qweb.add_template(utils.json_node_to_xml(templates));
+        var handleField = _.find(this.arch.children, function(c) {
```

Comment:
i'd only activate the d&g feature if widget="handle" is specified
```
        var handleField = _.findWhere(this.state.fieldsInfo.kanban, {widget: 'handle'});
        this.handleField = handleField && handleField.name;
```

Reply:
@aab-odoo The reason why I support sequence field because if you group kanban and do drag and drop record in same column then it calls resequence which do resequence on sequence field by default, so to have same behaviour like if handle field is not defined then it will do resequence on sequence field I kept this.